clients/python/tests/conftest.py

```
@@ -317,3 +339,85 @@ def mock_override(base_image, dest_dir, params):
+        artifact_id = resp.json()["id"]
+        try:
+            yield artifact_id
+        finally:
+            del_url = f"{REGISTRY_URL}/api/model_registry/v1alpha3/artifacts/{artifact_id}"
+            try:
+                requests.delete(del_url, headers=auth_headers, timeout=DEFAULT_API_TIMEOUT)
+            except Exception as e:
+                print(f"Failed to delete artifact {artifact_id}: {e}")
+    return _artifact_resource
+
+@pytest.fixture
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)